django/river/api/views.py

```diff
@@ -76,11 +74,10 @@ def create(self, request, *args, **kwargs):
 
 class ScriptsEndpoint(generics.ListAPIView):
```

Comment:
Not really related to this PR so we can do it later but I think that we should have a route to list cleaning scripts and another one to list merging scripts because they're not used at the same place in pyrog
(I can write the issue if you want)

Reply:
agreed, or maybe a filter in url params. I'll create an issue.